src/train.py

```diff
@@ -482,6 +618,21 @@ def train(args):
     model.eval()
     return model
 
+
+def log_tb_example(model, ex, step, name="train"):
+    model.eval()
+    if WRITER:
+        ex_mel = torch.tensor(ex["mel"]).unsqueeze(0)
+        ex_text = torch.LongTensor(ex["text"]).unsqueeze(0)
+        text_pred = model.asr(None, ex_mel.to(DEVICE), infer=True).squeeze(dim=0).detach().cpu().numpy()
+        speech_pred, speech_stop_pred = model.tts(ex_text.to(DEVICE), None, infer=True)
+        speech_pred = speech_pred.squeeze(dim=0).detach().cpu().numpy()
+        WRITER.add_text(f"{name}/text_gold", sequence_to_text(ex["text"]), step)
```

Comment:
We should trim this to length if it is not all ready, same with speech prediction!